aurora/main.tf

```diff
@@ -9,6 +9,7 @@ resource "aws_db_subnet_group" "aurora" {
 }
 
 resource "aws_db_parameter_group" "aurora_mysql" {
+  count       = "${length(var.instance_parameter_group_name) == 0 ? 1 : 0}"
```

Comment:
why is this change needed for the engine version?

Reply:
basically we don't create a mysql parameter group for mysql if another one is specified